src/services/authService.js

```diff
@@ -38,52 +35,43 @@ class AuthService {
     this.firebase.initializeApp(this.config);
   }
 
-  loadUserProfile() {
-    return new Promise((resolve) => {
-      this.keycloak
-        .loadUserProfile()
-        .then((profile) => {
-          resolve(profile);
-        })
-        .catch(function () {
-          alert("Failed to load user profile");
-        });
-    });
-  }
-
   login = () => {
-    cfaSignIn("google.com").subscribe();
+    cfaSignIn("google.com").subscribe((user)=>{
+     this.setSession(user);
+    });
   };
 
   logout = () => {
-    cfaSignOut().subscribe();
-    this.setSession(null);
+    cfaSignOut().subscribe(()=>{
+     this.removeSession();
+    });
+    
   };
-
-  setSession = (accessToken) => {
-    if (accessToken) {
-      localStorage.setItem("accessToken", accessToken);
-      axios.defaults.headers.common.Authorization = `Bearer ${accessToken}`;
-    } else {
+  
+  
+  removeSession(){
       localStorage.removeItem("accessToken");
+      localStorage.removeItem("userDataKey");
```

Comment:
accessToken looks very common name, the chances are it may get overriden by another website (very less chance).
So change it's name to `cfcAccessToken`

Reply:
Sure @Abhishek-kumar09 I'll change that but as far as I know, the local storage is shared between all tabs and windows of the same origin even if the keys are the same it shouldn't create a problem, correct me if I'm wrong @Abhishek-kumar09 :)